cli/wallet/wallet.go

```diff
@@ -966,31 +977,54 @@ func ReadWalletConfig(configPath string) (*config.Wallet, error) {
 	return cfg, nil
 }
 
-func newAccountFromWIF(w io.Writer, wif string, scrypt keys.ScryptParams) (*wallet.Account, error) {
+func newAccountFromWIF(w io.Writer, wif string, scrypt keys.ScryptParams, label *string, pass *string) (*wallet.Account, error) {
+	var (
+		phrase, name string
```

Comment:
I'm not sure why you have both label/pass and phrase/name.

Reply:
It's done to distinguish empty label/pass from the missing one. Do we need to remove `phrase` and `name`?

Reply:
You can't easily do `*pass, err = input.ReadPassword(...)`, that'd be the most convenient thing, so it's either phrase/name or `pass = &ret_from_readpwd`. OK, let's have it as is.

Reply:
```
You can't easily do *pass, err = input.ReadPassword(...)
```
Yes, that was the reason for keeping it this way.

Reply:
Reminds me of #2671, btw. This would be easier to handle with `[]byte`.